syntax = "proto3";
package block_external.v1;

import "google/protobuf/timestamp.proto";
message Block {
  bytes cur_hash = 1;
  bytes prev_hash = 2;
  google.protobuf.Timestamp unix_timestamp = 3;
  message Key {
    uint64 x = 1;
    uint64 y = 2;
  }
  Key mined_by = 4;
  uint64 ledger_id = 5;
  uint64 mining_points = 6;
  double luck = 7;
  bytes contained_data = 8;
}
